samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs

```diff
@@ -93,6 +79,11 @@ public void Start()
         }
     }
 
+#if UNITY_IOS
```

Comment:
Best to throw below the non P/Invoke methods (so we can read through `InitSentry` and others before finding the helper methods